.travis.yml

```diff
@@ -1,23 +1,21 @@
-branches:
-  only:
-    - master
```

Comment:
why do you remove it?

Reply:
I mean does it correct to report to sonarqube from any forks\branches?

Reply:
It's add possibility to build and perform code analysis for all branches.
And as I understood it necessary to process PRs.